test/acceptance/test/testsuite_common_test.go

```diff
@@ -3,13 +3,15 @@
 package acceptance
 
 import (
+	"fmt"
 	"os"
 	"testing"
 
 	. "github.com/onsi/ginkgo"
 	ginkgo "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 	log "github.com/sirupsen/logrus"
+	"github.com/weaveworks/weave-gitops/pkg/status"
```

Comment:
Could this be avoided, given it’s a black box test, should avoid using internal functions/components.